samples/switchless/README.md

```diff
@@ -1,13 +1,14 @@
 # Switchless Calls Sample
 
-This sample demonstrates how to make switchless calls to host from inside an enclave.
+This sample demonstrates how to make switchless ocalls to host from inside an enclave, and switchless ecalls to enclave
```

Comment:
grammar: "to host" -> "to a host application"
"to enclave" -> "to an enclave"

Reply:
Fixed.